purchases/src/main/kotlin/com/revenuecat/purchases/AppLifecycleHandler.kt

```diff
@@ -3,17 +3,31 @@ package com.revenuecat.purchases
 import androidx.lifecycle.DefaultLifecycleObserver
 import androidx.lifecycle.LifecycleOwner
 
+@SuppressWarnings("EmptyFunctionBlock")
 internal class AppLifecycleHandler(
     private val lifecycleDelegate: LifecycleDelegate,
 ) : DefaultLifecycleObserver {
+    // Some functions are implemented and left empty to prevent issues with Java default interfaces
+
+    override fun onCreate(owner: LifecycleOwner) {
+    }
```

Comment:
maybe we can group the ones that are empty together and add a comment + a region for clarity?